### DISCUSSION TITLE
splitting a compressed folder into parts

### DISCUSSION THREAD

**Discussion by sidielnikov** [samuel-lucas6/Kryptor#47] [Ideas]
hello, since folders are now being compressed, I suggest adding the function of splitting the compressed folder into parts, for example, in order to make it more convenient to upload to the cloud (this function is implemented in picocrypt)

**Comment by samuel-lucas6** [comment]
I'm not sure how that would be done. How would you know the order of the chunks to recombine them? Something similar can already be done by encrypting subdirectories instead of parent directories.

**Reply by sidielnikov** [reply]
One option to use the Shamir Secret Sharing algorithm or to do it as https://github.com/hacique/picocrypt. If you encrypt subcatals, then it will be a very long time
